label_sync/labels.yaml

```diff
@@ -1814,6 +1814,144 @@ repos:
         target: both
         prowPlugin: label
         addedBy: anyone
+      - color: 0052cc
+        description: Issues or PRs related to CAPBK
+        name: area/kubeadm-bootstrap
```

Comment:
I think this should be bootstrap-kubeadm and below control-plane-kubeadm 
```yaml
➜ k get provider -A
NAMESPACE                           NAME                    AGE   TYPE                     PROVIDER      VERSION
capd-system                         infrastructure-docker   37m   InfrastructureProvider   docker        v1.4.99
capi-kubeadm-bootstrap-system       bootstrap-kubeadm       43m   BootstrapProvider        kubeadm       v1.4.99
capi-kubeadm-control-plane-system   control-plane-kubeadm   43m   ControlPlaneProvider     kubeadm       v1.4.99
capi-system                         cluster-api             43m   CoreProvider             cluster-api   v1.4.99
```

I wonder if "provider" would be helpful as well (also for infrastructure-docker)

Maybe

area/provider/infrastructure-docker
area/provider/bootstrap-kubeadm
area/provider/control-plane-kubeadm
area/provider/core

(not sure if we don't like the provider segment)

or: 

area/infrastructure-provider-docker
area/bootstrap-provider-kubeadm
area/control-plane-provider-kubeadm
area/core-provider

Reply:
On changing the label naming here - to add "provider" either as a path prefix or elsewhere - I don't think it really helps or hinders. Just happy to pick any form and stick to it.

Agree with moving stuff into a core folder, but this probably isn't the place to have a good discussion on it!

Reply:
I would prefer if we have provider in the label name. If we don't want an additional segment we can use the form without, otherwise I would pick the ones in this format: "area/provider/infrastructure-docker"

Definitely just wanted to bring up the topic :)

Reply:
Added `area/provider/core`
updated labels as 
```
area/provider/infrastructure-docker
area/provider/bootstrap-kubeadm
area/provider/control-plane-kubeadm
```